docs/src/pages/components/rating/SimpleRating.tsx

```diff
@@ -27,8 +27,8 @@ export default function SimpleRating() {
         <Rating name="disabled" value={value} disabled />
       </Box>
       <Box component="fieldset" mb={3} borderColor="transparent">
-        <Typography component="legend">Pristine</Typography>
-        <Rating name="pristine" value={null} />
+        <Typography component="legend">no rating given</Typography>
```

Comment:
```suggestion
        <Typography component="legend">No rating given</Typography>
```

For harmony with the other labels?

<img width="176" alt="Capture d’écran 2020-09-16 à 12 10 26" src="https://user-images.githubusercontent.com/3165635/93324005-9ec60280-f815-11ea-9b74-31108dd36a7a.png">

Reply:
I am a bit unsure about the wording so I was hoping it wouldn't be final. But I think I'm now OK with the verbosity. Though I would've preferred perfect alignment 😄

Reply:
It's great like this 👌